cmake/configs/default.cmake

```diff
@@ -40,12 +40,7 @@ hunter_config(Async++ VERSION 0.0.3-hunter)
 hunter_config(Avahi VERSION 0.6.31)
 hunter_config(Beast VERSION 1.0.0-b84-hunter-0)
 hunter_config(BZip2 VERSION 1.0.6-p3)
-if(MSVC OR MINGW)
```

Comment:
1. I don't think it should affect MinGW (no vcvarsall.bat usage)
2. Let's update build scheme, then default versions

Reply:
Okey I updated PACKAGE_INTERNAL_DEPS_ID
I did it before but the merge masked it

Reply:
> I did it before but the merge masked it

I wasn't masked, the scheme was updated with https://github.com/ruslo/hunter/commit/67ebd75b93c07a03503635919cbb63f7c6b85291